docker/owlbot/nodejs/cloudbuild.yaml

```diff
@@ -1,10 +1,18 @@
 steps:
-    # Write the latest commit message to a file in the workspace.
-    - name: 'gcr.io/cloud-builders/git'
-      entrypoint: '/bin/sh'
-      args:
-        - '-c'
-        - 'git log -1 --format="%s%n%n%b%nSource-Link: https://github.com/googleapis/synthtool/commit/%H" > post-processor-changes.txt'
+  # Write the latest commit message to a file in the workspace.
+  - name: 'gcr.io/cloud-builders/git'
+    entrypoint: '/bin/sh'
+    args:
+      - '-c'
+      # Get the last commit in synthtool and write it to post-processor-changes.txt.
+      # If the commit message ends with parenthesis, remove them.
+      # If the commit message starts with feat or fix, replace it with chore.
+      # If the commit message includes ! followed by a colon to signal a breaking change, remove the !.
+      - >
+          git log -1 --format="%B%n%nSource-Link: https://github.com/googleapis/synthtool/commit/%H" > post-processor-changes.txt &&
+          sed -i "s/([^()]*)$//g" post-processor-changes.txt &&
```

Comment:
Do the opening/closing parentheses need escaping? Or does `sed` have non-regex match replacement rules?

Reply:
I dunno.  I copied the code from here:
https://github.com/googleapis/synthtool/blob/50db768f450a50d7c1fd62513c113c9bb96fd434/docker/owlbot/python/cloudbuild.yaml#L3-L15

It's been working for over a year.  I'd rather not change it because it's working.